_emails/us/michigan/royaloak.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Royal Oak, MI
+permalink: "/lasvegas"
+name: Letter to the City Commission and City Manager
+state: MI
+city: Royal Oak
+layout: email
+recipients:
+- ccfournier@romi.gov
+- ccdubuc@romi.gov
+- ccgibbs@romi.gov
+- cclevasseur@romi.gov
+- ccparuch@romi.gov
+- ccmacey@romi.gov
+- ccdouglas@romi.gov
+- citymanager@romi.gov
+subject: A budget for the community well-being of Royal Oak
+body: |-
+  To the City of Royal Oak's 50th Commission and City Manager,
+
+  My name is [YOUR NAME], and I am a resident of Royal Oak. This past week, our nation has been gripped by protests calling for a rapid and meaningful reconsideration of the role of policing in communities as well as an end to racism and anti-Blackness in America. As the Commission prepares the budget for the 2021 fiscal year beginning July 1, 2020, I demand that you reconsider the budget of the Royal Oak Police Department (ROPD).
+
+  The fourth goal of the ROPD listed on the 2020-2021 Recommended Budget is "to provide exceptional public safety for major events, such as Woodward Dream Cruise and Arts, Beats and Eats." Given that both of those events have been canceled it is reasonable to ask that the $61,000 that has been budgeted for these events be reallocated to sectors that could more effectively benefit Royal Oak, such as community development, health and welfare, affordable housing, the commissions' sustainability initiative, public works, and education.
```

Comment:
Just want to confirm the budget amounts are correct - where is the 61,000 from? From what I understand on the recommended budget's pdf, 88,770 total expenditures are allotted for Woodward Dream Cruise and Arts (page 149) and 43,270 total expenditures are allotted for Beats and Eats (page 150). If I have misunderstood then please ignore.

Reply:
Hi! The 61,000 is taken from $22,230 in personnel services for Arts, Beats and Eats (pg 150) and $38,770 in personnel services for the Woodward Dream cruise on page 149

Reply:
But I agree that number should reflect the total expenditures